app/controllers/SessionsController.scala

```diff
@@ -81,6 +91,8 @@ class SessionsController @Inject()(messagesApi: MessagesApi,
 
   implicit val knolxSessionInfoFormat: OFormat[KnolxSession] = Json.format[KnolxSession]
   implicit val sessionSearchResultInfoFormat: OFormat[SessionSearchResult] = Json.format[SessionSearchResult]
+  implicit val categoriesFormat: OFormat[CategoryInfo] = Json.format[CategoryInfo]

```

Comment:
Remove (view) dependency on model case class and create/use a separate case class instead.